widget/ui/src/components/BottomLogo/BottomLogo.tsx

```diff
@@ -1,48 +1,34 @@
-import React from 'react';
 import { i18n } from '@lingui/core';
-import { styled } from '../../theme';
-import { Typography } from '../Typography';
-
-const Container = styled('div', {
-  display: 'flex',
-  width: '100%',
-  justifyContent: 'end',
-  alignItems: 'center',
-});
+import React from 'react';
 
-const Logo = styled('svg', {
-  fill: '$foreground',
-  width: '$24',
-  margin: '0 $8 0 $16',
-});
```

Comment:
Please move logo to a separate component and use divider component instead of margin